CMakeLists.txt

```diff
@@ -85,6 +85,26 @@ endif(WITH_CCACHE)
 
 option(WITH_SCCACHE "Build with sccache.")
 if(WITH_SCCACHE)
+  if(NOT FORCE_MAX_JOBS)
```

Comment:
if this is a new setting, might want to add it as a cached variable, like
```cmake
set(FORCE_MAX_JOBS ... "<doc string>")
```

Reply:
IIUC, `NINJA_MAX_COMPILE_JOBS` and `NINJA_MAX_LINK_JOBS` override this setting? so probably we should also document this in `FORCE_MAX_JOBS` 's doc string?

Reply:
I dropped `FORCE_MAX_JOBS` in favor of internally setting the `NINJA_` variables; that way we're not introducing any new settings.